src/python/pants/backend/jvm/ivy_utils.py

```diff
@@ -533,8 +536,11 @@ def __init__(self, org, name, rev, classifier=None, ext=None):
   def __eq__(self, other):
     return isinstance(other, IvyModuleRef) and self._id == other._id
 
-  def __ne__(self, other):
-    return not self == other
+  # TODO(python3port): Return NotImplemented if other does not have attributes
+  def __lt__(self, other):
```

Comment:
Replaced `__cmp__` with rich comparisons, which are auto-generated from the `@total_ordering` annotation.

Same semantics as before